duo_wordpress.php

```diff
@@ -378,6 +383,15 @@ function duo_settings_xmlrpc() {
         echo "<input id='duo_xmlrpc' name='duo_xmlrpc' type='checkbox' value='off' $val /> Yes<br />";
         echo "Using XML-RPC bypasses two-factor authentication and makes your website less secure. We recommend only using the WordPress web interface for managing your WordPress website.";
     }
+    function duo_settings_rest() {
+        $val = '';
+        if(duo_get_option('duo_rest', 'off') == 'off') {
```

Comment:
Also, I think this needs to be updated to match the other fetch for `duo_get_option`. In other words, this should be disabled by default.